src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1973,7 +1973,7 @@ MavlinkReceiver::handle_message_rc_channels_override(mavlink_message_t *msg)
 	// metadata
 	rc.timestamp = hrt_absolute_time();
 	rc.timestamp_last_signal = rc.timestamp;
-	rc.rssi = RC_INPUT_RSSI_MAX;
+	rc.rssi = input_rc_s::RC_RSSI_MAX;
```

Comment:
```suggestion
	rc.rssi = input_rc_s:: RC_RSSI_UNDEFINED;
```

Would undefined be more accurate? Not critical either way.

Reply:
@dagar done here: https://github.com/PX4/PX4-Autopilot/pull/16042/commits/caec7e340fb7fffe1c01d6771d31698cfbd7fcb6#